bullet_train/docs/super-scaffolding/options.md

```diff
@@ -0,0 +1,24 @@
+# Super Scaffolding Options
+
+There are different flags you can pass to the Super Scaffolding command which gives you more flexibility over creating your model. Add the flag of your choice to **the end** of the command for the option to take effect.
+```
+bin/super-scaffold crud Project Team description:text_field --sortable
+```
+
+Most of these include skipping particular functionalities, so take a look at what's available here and pass the flag that applies to your use-case.
+
+| Option | Description |
+|--------|-------------|
+| `--sortable` | [Details here](/docs/super-scaffolding/sortable.md) |
+| `--namespace=customers` | [Details here](/docs/namespacing.md) |
+| `--navbar="ti-world"` | Pass the Themify icon or Font Awesome icon of your choice to automatically add it to the navbar* |
```

Comment:
@gazayas What do you think about changing this to the currently `sidebar` syntax so that we can merge this without needing to wait on the other `sidebar` => `navbar` PRs. (Those are going to need coordination with a new release of `core` and I don't know when that's going to be.)

Reply:
@jagthedrummer Works for me, just went ahead and changed it.